src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -2248,12 +2263,31 @@ private static IEnumerable<object[]> ToLower_Culture_TestData()
             yield return new object[] { "H\u0131 World", "h\u0131 world", CultureInfo.InvariantCulture };
         }
 
-        [Theory]
-        [MemberData(nameof(ToLower_Culture_TestData))]
-        public static void ToLower_Culture(string actual, string expected, CultureInfo culture)
+        [Fact]
+        public static void Test_ToLower_Culture()
+        {
+            RemoteInvoke(() =>
+            {
+                foreach (var testdata in ToLower_Culture_TestData())
+                {
+                    CultureInfo originalCulture = CultureInfo.CurrentCulture;
+                    try
+                    {
+                        CultureInfo.CurrentCulture = (CultureInfo)testdata[2];
+                        ToLower_Culture((string)testdata[0], (string)testdata[1]);
+                    }
+                    finally
+                    {
+                        CultureInfo.CurrentCulture = originalCulture;
+                    }
+                }
+                return SuccessExitCode;
+            }).Dispose();
```

Comment:
In this case, CurrentCulture is an input coming from test data and changes for each case.